subsystems/data_destination.go

```diff
@@ -11,6 +11,9 @@ import (
 // Do not use it.
 // You have been warned.
 type DataDestination interface {
+	// Selector returns the last known selector for the data store.
```

Comment:
Consider adding that the selector may be unknown.  Is that the `fdv2proto.NoSelector()`?